src/compiler/getAppSource.ts

```diff
@@ -48,30 +56,52 @@ function makeICompilerFileMap(compilerFiles: ICompilerFile[]): IMapCompilerFile
         .reduce((acc: IMapCompilerFile, curr: IMapCompilerFile) => ({ ...acc, ...curr }), {});
 }
 
+async function getTSConfig(projectPath: string): Promise<TSConfig> {
+    const tsconfigFile = await fs.readFile(join(projectPath, 'tsconfig.json'));
+
+
+    if (!tsconfigFile) {
+        logger.debug('Project tsconfig.json file not found - ignoring');
+
+        return {};
+    }
+
+    try {
+        const a = JSON.parse(tsconfigFile.toString()) as TSConfig;
+        return a;
+    } catch {
+        logger.warn('Invalid tsconfig.json file - ignoring');
```

Comment:
Please, let's not ignore the file if it throws any error.

Reply:
We **need** to ignore the tsconfig file if it is invalid, we can't risk breaking backward compatibility